audisp-tacplus.c

```diff
@@ -501,7 +385,7 @@ static void get_acct_record(auparse_state_t *au, int type)
         taskno = (uint16_t) pid;
     }
     else /* should never happen, if it does, records won't match */
-        taskno = tac_magic();
+        taskno = (u_int32_t)rand();
```

Comment:
>rand [](http://example.com/codeflow?start=28&length=4)

Is it a bug fix?

Reply:
Similer with xcalloc, the upstream project using tac_magic from a customized version of libpam_tacplus:
https://github.com/daveolson53/pam_tacplus/blob/1daa4e34db9de5cb28538b9af7ef2c213886c655/libtac/lib/magic.h

But SONiC version of libpam_tacplus not have that method, so we use rand to generate random seed fir session ID.